core/server/common/src/main/java/alluxio/cli/ValidateEnv.java

```diff
@@ -300,5 +349,30 @@ public static void main(String[] args) throws InterruptedException {
     System.exit(validate(args));
   }
 
+  /**
+   * Parses the command line arguments and options in {@code args}.
+   *
+   * After successful execution of this method, command line arguments can be
+   * retrieved by invoking {@link CommandLine#getArgs()}, and options can be
+   * retrieved by calling {@link CommandLine#getOptions()}.
+   *
+   * @param args command line arguments to parse
+   * @return {@link CommandLine} object representing the parsing result
+   * @throws InvalidArgumentException if command line contains invalid argument(s)
+   */
+  private static CommandLine parseArgsAndOptions(Options options, String... args)
```

Comment:
looks like the same code block as alluxio.cli.AbstractCommand.parseAndValidateArgs. can it be re-used?

Reply:
That method is in `AbstractCommand` that is logically a different component. Do we want to move the code block to a common `Utils` component?

Reply:
sg